modules/renter/hostdb/hostdb.go

```diff
@@ -58,5 +118,7 @@ func New() *HostDB {
 	}
 	go hdb.threadedScan()
 
-	return hdb
+	cs.ConsensusSetSubscribe(hdb)
```

Comment:
yeah, I wasn't sure about this. But it's the hostdb that processes consensus changes, not the renter.